libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -235,7 +235,7 @@ void AC_AttitudeControl::input_quaternion(Quaternion& attitude_desired_quat, Vec
         // and an exponential decay specified by _input_tc at the end.
         _ang_vel_target.x = input_shaping_angle(wrap_PI(attitude_error_angle.x), _input_tc, get_accel_roll_max_radss(), _ang_vel_target.x, ang_vel_target.x, radians(_ang_vel_roll_max), _dt);
         _ang_vel_target.y = input_shaping_angle(wrap_PI(attitude_error_angle.y), _input_tc, get_accel_pitch_max_radss(), _ang_vel_target.y, ang_vel_target.y, radians(_ang_vel_pitch_max), _dt);
-        _ang_vel_target.z = input_shaping_angle(wrap_PI(attitude_error_angle.z), _input_tc, get_accel_yaw_max_radss(), _ang_vel_target.z, ang_vel_target.z, radians(_ang_vel_yaw_max), _dt);
+        _ang_vel_target.z = input_shaping_angle(wrap_PI(attitude_error_angle.z), 3.0f * _rate_y_tc, get_accel_yaw_max_radss(), _ang_vel_target.z, ang_vel_target.z, radians(_ang_vel_yaw_max), _dt);
```

Comment:
because this method wants a time constant for attitude and not rate.  I've found that 3 times the time constant for rate is a good time constant for attitude.  I don't want another time constant for yaw attitude.  I didn't think the input_tc should be used for the yaw axis but I could be wrong.

Reply:
3 times puts the default `PILOT_RATE_Y_TC` (0.05) exactly to the default of `ATC_INPUT_TC` of 0.15. So if both are default this change does not do anything. If someone had set `ATC_INPUT_TC` this is a change in behavior. 

I would be tempted to leave off these changes replacing  `_input_tc` for a future PR.